src/action_creators.js

```diff
@@ -0,0 +1,47 @@
+import axios from "axios"
+// helpers
+function urlTagStringToUrl (urlTagString) { return urlTagString.trim().replace(/^<url>/, "").replace(/<\/url>$/, "") }
```

Comment:
I always favor conciseness and readability - your sourcemaps will still
give yo good enogh debug info to know it is the fn expression causing the
issue as opposed to the named. If you keep named fn then at least make it
multiline

On Sunday, February 14, 2016, mLuby <notifications@github.com> wrote:

> In src/action_creators.js
> <https://github.com/mLuby/cats-test/pull/1#discussion_r52849310>:
>
> > @@ -0,0 +1,47 @@
> > +import axios from "axios"
> > +// helpers
> > +function urlTagStringToUrl (urlTagString) { return urlTagString.trim().replace(/^<url>/, "").replace(/<\/url>$/, "") }
>
> I weighed pros and cons of function vs => and here's what I have:
> function > =>:
>
>    - named function aids debugging
>    - unambiguously a function rather than a value
>    - hoisted
>
> => > function:
>
>    - shorter
>    - implicit return
>    - can be curried
>
> So I'm leaning slightly toward function when it's a top-level definition.
> What do you think @gabdallah222 <https://github.com/gabdallah222> ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mLuby/cats-test/pull/1/files#r52849310>.
>